server/src/routes/comments.js

```diff
@@ -23,4 +23,19 @@ router.post('/', async (req, res) => {
   }
 });
 
+// Update one comment
+// eslint-disable-next-line no-unused-vars
+router.put('/:id', async (req, res) => {
+  try {
+    if (req.body.body != null) {
```

Comment:
Awesome job, just one comment: Is it possible to change the request attribute name - `req.body.body` to something like `req.body.commentBody`? It just helps with readability :)

Reply:
done :)